front/components/assistant/AssistantPicker.tsx

```diff
@@ -62,51 +70,57 @@ export function AssistantPicker({
             isSelect
             size={size}
             tooltip="Pick an agent"
+            disabled={isLoading}
           />
         )}
       </DropdownMenuTrigger>
-      <DropdownMenuContent className="min-w-[300px]">
-        <DropdownMenuSearchbar
-          ref={searchbarRef}
-          placeholder="Search"
-          name="input"
-          value={searchText}
-          onChange={setSearchText}
-          onKeyDown={(e) => {
-            if (e.key === "Enter" && searchedAssistants.length > 0) {
-              onItemClick(searchedAssistants[0]);
-              setSearchText("");
-              close();
-            }
-          }}
-        />
-        <DropdownMenuSeparator />
-        <ScrollArea className="flex max-h-[300px] flex-col" hideScrollBar>
-          {searchedAssistants.map((c) => (
-            <DropdownMenuItem
-              key={`assistant-picker-${c.sId}`}
-              icon={() => <Avatar size="xs" visual={c.pictureUrl} />}
-              label={c.name}
-              onClick={() => {
-                onItemClick(c);
+      <DropdownMenuContent className="w-96" align="end">
+        <div className="flex gap-1.5 p-1.5">
```

Comment:
Don't we want to extend DropdownMenuSearchbar to support button ? Otherwise maybe we should get rid of it ?